api/envoy/extensions/filters/http/composite/v3/composite.proto

```diff
@@ -57,4 +58,11 @@ message ExecuteFilterAction {
   // Only one of ``typed_config`` or ``dynamic_config`` can be set.
   DynamicConfig dynamic_config = 2
       [(udpa.annotations.field_migrate).oneof_promotion = "config_type"];
+
+  // Percentage of traffic the action filter will be executed. By default this is 100%.
+  // This is to support sampling behavior for certain filters like ext_proc, access_logging ...
+  // For example, if ``sample_percent`` is configured with 30%, a dice roll with that
+  // probability is done. The underline action filter will only be executed if the
+  // dice roll returns positive. Otherwise, the action filter is skipped.
+  type.v3.FractionalPercent sample_percent = 3;
```

Comment:
I would use `core.v3.RuntimeFractionalPercent` so this can be controlled via Runtime as well if desired.

Reply:
Good idea, done!